core/fetcher_no_race_test.go

```diff
@@ -15,7 +15,7 @@ import (
 // TestBlockFetcherHeaderValues tests that both the Commit and ValidatorSet
 // endpoints are working as intended.
 func TestBlockFetcherHeaderValues(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
```

Comment:
Did these timeout bumps helped anyhow?

Reply:
this change actually did help. In CI and locally I was observing the test time out at 10 seconds but it actually took ~11 seconds to complete so I bumped to 15 seconds so that it had enough time to complete.